Makefile

```diff
@@ -44,6 +44,7 @@ DOCKER_CMD     = docker run --rm -ti -v $(PWD):/go/src/$(SC_PKG) \
 
 ifneq ($(origin DOCKER),undefined)
   # If DOCKER is defined then make it the full docker cmd line we want to use
+  USE_DOCKER=DOCKER=1
```

Comment:
It is not obvious to me what the purpose of `USE_DOCKER` is that isn't already served by `DOCKER`.  Enlighten me?